Sources/TensorFlow/Layers/Sequential.swift

```diff
@@ -0,0 +1,240 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// A layer that sequentially composes two other layers.
+public struct Sequential<Layer1: Layer, Layer2: Layer>: Layer where
```

Comment:
For consistency with the rest of the code base, put `where` on the following line.

Reply:
Done.